### PR TITLE
com.github.alexkdeveloper.notepad 1.0.0

### PR DESCRIPTION
**Pull request by alexkdeveloper**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.github.alexkdeveloper.notepad

**Comment by hfiguiere** [comment]
PR should be against `new-pr`

File: com.github.alexkdeveloper.notepad.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.github.alexkdeveloper.notepad",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.alexkdeveloper.notepad",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
```
Comment (hfiguiere) [inline com.github.alexkdeveloper.notepad.json:10]: don't you need DRI too?

Comment (alexkdeveloper) [reply com.github.alexkdeveloper.notepad.json:10]: What for? The app is already working well)

**Comment by alexkdeveloper** [comment]
What is the problem? The program is working. There are no extra permissions in the manifest.

**Comment by barthalion** [comment]
bot, build com.github.alexkdeveloper.notepad

**Comment by barthalion** [comment]
What does `save note` button do? Are files saved somewhere on the filesystem, or it just stores changes to the note somewhere in XDG?

**Comment by alexkdeveloper** [comment]
The storage path of notes is defined in this way:
`directory_path = Environment.get_user_data_dir()+"/notes_for_notepad_app";`
https://valadoc.org/glib-2.0/GLib.Environment.get_user_data_dir.html

**Pull request closed without merging**